applications/temporalgrid-demo/src/popup.js

```diff
@@ -0,0 +1,41 @@
+import { Popup } from '@globalfishingwatch/react-map-gl';
+import React from 'react';
+
+function PopupWrapper ({ tooltipEvent, closeButton, closeOnClick, className, onClose, children }) {
```

Comment:
I think its worth having this component either react-map-gl custom fork or ui-components to be able to consume hoverTooltipEvent as simple as possible.